suite-native/formatters/src/components/CryptoToFiatAmountFormatter.tsx

```diff
@@ -29,5 +31,12 @@ export const CryptoToFiatAmountFormatter = ({
         cryptoValue: value,
     });
 
-    return <FiatAmountFormatter network={network} value={fiatValue} {...textProps} />;
+    return (
+        <FiatAmountFormatter
+            network={network}
+            value={fiatValue}
+            isForcedDiscreetMode={isForcedDiscreetMode}
```

Comment:
```suggestion
```

the `isForcedDiscreetMode` prop will be passed automatically via the `...textProps`

Reply:
good point! 4a845d4e24